css/src/atomics/width.scss

```diff
@@ -0,0 +1,101 @@
+$widths: 100, 150, 200, 250, 300, 350, 400, 450, 500 !default;
```

Comment:
Another thought - I think that would be the right place to add a `width-full` helper.

Oh, also it might be great to have `width-auto` 🤔

Reply:
yes, good idea

Reply:
What about the full width class? I do see quite a lot of usage of similar class in docs

Reply:
Might be better in the future to come up with a .column friendly scheme for percentage widths, but for now, pushing up only `.width-full` and the same for tablet and desktop.

Reply:
Don't forget to add it to the doc page!